src/rpft/rapidpro/models/common.py

```diff
@@ -100,5 +100,5 @@ def assign_uuid(self, uuid_dict):
     def render(self):
         render_dict = {"name": self.name, "uuid": self.uuid}
         if self.query:
-            render_dict["query"] = query
+            render_dict["query"] = self.query
```

Comment:
@geoo89 I want to make sure this is what was originally intended. Is it a safe change?

Reply:
Yes, this looks like a bug that you just fixed.